usr/src/lib/libcmdutils/common/taskq.c

```diff
@@ -245,11 +265,11 @@ taskq_create(const char *name, int nthreads, pri_t pri,
 		ASSERT3S(nthreads, >=, 1);
 	}
 
-	rw_init(&tq->tq_threadlock, NULL, RW_DEFAULT, NULL);
-	mutex_init(&tq->tq_lock, NULL, MUTEX_DEFAULT, NULL);
-	cv_init(&tq->tq_dispatch_cv, NULL, CV_DEFAULT, NULL);
-	cv_init(&tq->tq_wait_cv, NULL, CV_DEFAULT, NULL);
-	cv_init(&tq->tq_maxalloc_cv, NULL, CV_DEFAULT, NULL);
+	VERIFY0(rwlock_init(&tq->tq_threadlock, USYNC_THREAD, NULL));
+	VERIFY0(mutex_init(&tq->tq_lock, USYNC_THREAD, NULL));
```

Comment:
This needs to have LOCK_ERRORCHECK as well, otherwise none of the verify statements will be useful.